k8s-daemonset/README.md

```diff
@@ -51,6 +51,14 @@ one is best for your use case.
 
 #### Daemonsets
 
+If cluster enable RBAC please setting up the permission:
```

Comment:
This same information is capture in the RBAC section further down this file. What do you think about moving that section up to here, and combining it with what you have documented here?